Gopkg.toml

```diff
@@ -5,3 +5,5 @@
 [[constraint]]
   name = "github.com/pborman/uuid"
   version = "1.1.0"
+  name = "github.com/docker/docker"
+  version = "1.13.1"
```

Comment:
/blocker or comment needed

I feel like we should have picked up a much newer version than this. Maybe dep is confused by the docker switch in versioning schemes.

I remember us discussing this at the time, but I can't remember if we specifically chose this or why.

Reply:
so, docker hasn't made a release in over a year.
most recent release is
https://github.com/moby/moby/releases/tag/v17.03.2-ce from jun 2017
Kubernetes master & 1.12-beta branch uses a9fbbdc8dd8794b20af358382ab780559bca589d from june 2018.

I'm quite shocked.
Perhaps we should look into using https://github.com/fsouza/go-dockerclient ?

Reply:
https://github.com/moby/moby/releases/tag/v1.13.1 is from feb 2017